packages/core/test/lib/base.test.ts

```diff
@@ -188,26 +221,31 @@ describe('BaseClient', () => {
   describe('captureException', () => {
     test('captures and sends exceptions', () => {
       const client = new TestClient({ dsn: PUBLIC_DSN });
+
       client.captureException(new Error('test exception'));
-      expect(TestBackend.instance!.event).toEqual({
-        environment: 'production',
-        event_id: '42',
-        exception: {
-          values: [
-            {
-              type: 'Error',
-              value: 'test exception',
-            },
-          ],
-        },
-        timestamp: 2020,
-      });
+
+      expect(TestBackend.instance!.event).toEqual(
+        expect.objectContaining({
```

Comment:
Why is this `expect.objectContaining` required? Is it related to the feature this PR introduces? Same questions for other edits below.

Reply:
Yes - it's that or add `processingMetadata` to all of those objects, but since `processingMetadata` has nothing to do with any of those tests, I'd rather not muddy the waters with its presence.